### PR TITLE
TabPanel: fix kbn after runtime loop activation and swipe

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js [hunk 1]
```diff
@@ -1121,6 +1121,26 @@ QUnit.module('keyboard navigation', {
         assert.equal(tabsFocusedIndex, $(this.instance.option('focusedElement')).index(), 'multiView focused element is equal tabs focused element');
     });
 
+    QUnit.test('should go back to first index when loop = true and root element is focused', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: Why do you add "root element is focused"? As far as I see focus is not important here, but important that we change loop at runtime and swipe before kbn
```suggestion
    QUnit.test('looping should work on keyboard navigation after loop runtime change to true and swipe', function(assert) {
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js [hunk 2]
```diff
@@ -1121,6 +1121,26 @@ QUnit.module('keyboard navigation', {
...
+        this.instance.option({
+            items: [1, 2, 3],
+            loop: false,
+            swipeEnabled: true,
+            focusStateEnabled: true
+        });
+        this.instance.option('loop', true);
+        this.$element.attr('tabindex', 0);
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: [synthetic test, crutch]
The broken scenario is reproduced in your playground even without crutches with manual tabindex setting, right?
It means that you just incorrectly do keyboard navigation in the test

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: [f38015f](https://github.com/DevExpress/DevExtreme/pull/28303/commits/f38015f174790a18cc848f8168582fdbd0f0db6d)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: [f38015f](https://github.com/DevExpress/DevExtreme/pull/28303/commits/f38015f174790a18cc848f8168582fdbd0f0db6d)

**Review by ksercs — DISMISSED** [review]

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**